### PR TITLE
Add rvm-windows to installation doc

### PR DESCRIPTION
**Pull request by magynhard**
Add the new RVM 4 Windows to the list of Ruby version managers at the installation documentation.

Included languages are `bg de en es fr id it ja ko pl pt ru tr zh_cn`

### CODE REVIEW

**Comment by vtamara** [comment]
Spanish is good.

File: ko/documentation/installation/index.md [hunk 1]
```diff
@@ -338,6 +338,14 @@ RVM은 다른 gemset도 관리해 줍니다.
 macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
 
 
+### RVM 4 Windows
+{: #rvm-windows}
+
+[RVM 4 Windows][rvm-windows]를 사용하면 Windows에 여러 개의 루비 설치를 
+설치하고 관리할 수 있습니다. 이 도구는 원본 RVM의 복제본으로, 원본 RVM과 
+동일한 명령줄 인터페이스를 제공하여 클래식 명령줄은 물론 파워쉘도 지원합니다.
```
Comment (marocchino) [inline ko/documentation/installation/index.md]:
```suggestion
[RVM 4 Windows][rvm-windows]를 사용하면 Windows에 여러 종류의 Ruby를 
설치하고 관리할 수 있습니다. 이 도구는 원본 RVM의 복제본으로, 원본 RVM과 
동일한 커맨드 라인 인터페이스를 제공하며 PowerShell은 물론 클래식 커맨드 라인도 지원합니다.
```

**Review by marocchino — APPROVED** [review]

File: ko/documentation/installation/index.md [hunk 2]
```diff
@@ -338,6 +338,14 @@ RVM은 다른 gemset도 관리해 줍니다.
 macOS, Linux나 다른 UNIX-계열 운영체제에서만 사용 가능합니다.
 
 
+### RVM 4 Windows
+{: #rvm-windows}
```
Comment (yous) [inline ko/documentation/installation/index.md:342]: `RVM 4 Windows` should be added to TOC between "RVM" and "uru."

Comment (magynhard) [reply ko/documentation/installation/index.md:342]: My idea was to make it analogous to "rbenv for windows", which also does not have its own entry in TOC, but follows "rbenv". So does "RVM 4 Windows" follow "RVM".

Comment (yous) [reply ko/documentation/installation/index.md:342]: Oh, I understand.

**Comment by magynhard** [comment]
Hey guys,

for me it looks like, that many languages are even not maintained anymore, as no more reviews happend within one month.

So, would you suggest me, to make a new PR only with Korean and English? Or is there a chance to get this PR ever merged without all languages reviewed?

**Comment by magynhard** [comment]
As there are lots of language reviewers not active anymore, this Pull Request will never succeed all reviews. So i decided to make a new one with English only and close this one.

New Pull Request with English only: https://github.com/ruby/www.ruby-lang.org/pull/3425

**Pull request closed without merging**